oio/cli/admin/cluster.py

```diff
@@ -140,13 +141,22 @@ def take_action(self, parsed_args):
 class ClusterUnlockAll(lister.Lister):
     """Unlock all services of the cluster"""
 
-    log = logging.getLogger(__name__ + '.ClusterUnlock')
+    log = logging.getLogger(__name__ + '.ClusterUnlockAll')
 
     def get_parser(self, prog_name):
-        return super(ClusterUnlockAll, self).get_parser(prog_name)
+        parser = super(ClusterUnlockAll, self).get_parser(prog_name)
+        parser.add_argument(
+            '-t', '--type',
```

Comment:
I would prefer positional arguments, as in `ClusterList` and `ClusterLocalList`.